### PR TITLE
update test addresses

### PR DESCRIPTION
**Pull request by Anmol1696**
Replace: #116 

## 1. Overview
Update addresses for test wallets as per the new cointype 118

### CODE REVIEW

File: contrib/local/configs/keys.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     {
       "name": "test1",
       "type": "local",
-      "address": "persistence1ht0tun4u5uj4f4z83p9tncjerwu27ycsm52txm",
+      "address": "persistence152nvc6f096v6n6tr5lg50xq22ak0chsr0ru7xc",
       "pubkey": {
         "@type": "/cosmos.crypto.secp256k1.PubKey",
         "key": "AoC0SFBrAyiHrtJ+uGFuxP9CyTK8zSppylBhB+KBONNq"
```
Comment (puneet2019) [inline contrib/local/configs/keys.json:9]: pubkeys should also be updated for all

File: contrib/local/configs/relayer-keys.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     {
       "name": "persistenceCore",
       "id": "core-1",
-      "hdpath": "m/44'/750'/0'/0/0",
+      "hdpath": "m/44'/118'/0'/0/0",
       "keys": [
         {
           "name": "rly-core0",
```
Comment (puneet2019) [inline contrib/local/configs/relayer-keys.json:30]: relayer addresses and pubkeys should also change in this file

File: contrib/local/configs/validators.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     {
       "name": "val1",
       "type": "local",
-      "address": "persistence1wh9z8dedh0mpcgzd5e2zqzsdu2j2d0zwfk3xz9",
+      "address": "persistence1fk57xyyxz89krc3nn5law2xekan84gufzagszw",
       "pubkey": {
         "@type": "/cosmos.crypto.secp256k1.PubKey",
         "key": "A3n4L7G8aFv7g3UfBZFVZA5oByH8H6E6oxBiKDpxrTPx"
```
Comment (puneet2019) [inline contrib/local/configs/validators.json:9]: pubkeys should also be updated.

**Review by puneet2019 — CHANGES_REQUESTED** [review]

File: contrib/local/configs/keys.json (hunk 1 above)
Comment (betterclever) [reply contrib/local/configs/keys.json:9]: @puneet2019  The pubkey should remain same post cointype change as well right?
I tried with the latest `persistenceCore` binary, for the same `mnemonic` the public key remains same but the address changes to what @Anmol1696 has updated it to. Is that a bug in the persistenceCore binary or indeed the intended behavior?

Comment (alienHunterOnMars) [reply contrib/local/configs/keys.json:9]: I checked and the pubKeys remain the same

File: contrib/local/configs/relayer-keys.json (hunk 2 above)
Comment (alienHunterOnMars) [reply contrib/local/configs/relayer-keys.json:30]: updated the addresses. pubkeys should not change

Comment (puneet2019) [reply contrib/local/configs/relayer-keys.json:30]: thanks, now it seems ok

Comment (puneet2019) [reply contrib/local/configs/relayer-keys.json:30]: now it seems ok, thanks!

File: contrib/local/configs/keys.json (hunk 1 above)
Comment (puneet2019) [reply contrib/local/configs/keys.json:9]: thanks! looks like a bug when keys were taken from gaia.. the pubkeys were not updated then

**Review by puneet2019 — APPROVED** [review]

**Review by betterclever — APPROVED** [review]

**Pull request merged**